server/src/main/java/org/apache/iotdb/db/mpp/plan/analyze/Analysis.java

```diff
@@ -195,6 +196,8 @@ public class Analysis {
 
   private OrderByParameter mergeOrderParameter;
 
+  private Ordering timeseriesOrderingForLastQuery = null;
```

Comment:
add some comments for this field, like: when there only exist order by timeseries clause in last query, this field will be set, otherwise it will be null.